test/e2e/suite/trustpolicy/trust_store.go

```diff
@@ -41,9 +41,7 @@ var _ = Describe("notation trust policy trust store test", func() {
 			artifact := GenerateArtifact("e2e-valid-signature", "")
 
 			notation.ExpectFailure().Exec("verify", "--allow-referrers-api", artifact.ReferenceWithDigest(), "-v").
-				MatchErrKeyWords("authenticity validation failed",
-					"truststore/x509/ca/invalid_store\\\" does not exist",
-					VerifyFailed)
+				MatchErrKeyWords("the trust store \"invalid_store\" of type \"ca\" does not exist")
```

Comment:
Will it always be of type "CA" that we test in this code, or could it be of another type  - signingAuthority ? should the error message include a note for both CA and signingAuthority ?

Reply:
This is the e2e test, it's just an example that's using the `CA` type. `signingAuthority` is supported.